### DISCUSSION TITLE
locales are loaded after response

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5411] [Error / Bug Report] [answered]
When loading a page in wiki.js first you'll see a locale string (most likely) and then a few seconds after it'll replace with the proper string.

expected behaviour:
locales are inserted while crafting response.

**Comment by NGPixel** [comment, marked as answer]
This only happens the very first time you load Wiki.js. All subsequent requests will use the cache from your browser localstorage.

**Comment by ghost** [comment]
So this is stored on client side?
Is there an option to get this rendered on server side?
